outputs.tf

```diff
@@ -82,3 +82,13 @@ output "private_route_table_ids" {
   value       = aws_route_table.private[*].id
   description = "IDs of the private route tables"
 }
+
+output "subnet_share_id" {
+  value       = var.share_public_subnets || var.share_private_subnets ? module.subnet_sharing[0].id : ""
+  description = "The ID of the subnet share in resource access manager if any"
+}
+
+output "subnet_share_arn" {
+  value       = var.share_public_subnets || var.share_private_subnets ? module.subnet_sharing[0].arn : ""
+  description = "The ARN of the subnet share in resource access manager if any"
+}
```

Comment:
Please add newline at the end of your files (noticed that in the subnet-sharing repo as well, but forgot to mention it)